tests/lib/schema_validation_test.py

```diff
@@ -170,6 +171,34 @@ def test_valid_without_auto_neg_and_missing_speed(self, default_data):
 
         libnmstate.validator.validate(default_data)
 
+    def test_valid_with_sriov_total_vfs(self, default_data):
+        default_data[Interface.KEY][0].update(
+            {SRIOV.CONFIG_SUBTREE: {SRIOV.TOTAL_VFS: 5}}
+        )
+        libnmstate.validator.validate(default_data)
+
+    def test_negative_total_vfs_is_invalid(self, default_data):
+        default_data[Interface.KEY][0].update(
+            {SRIOV.CONFIG_SUBTREE: {SRIOV.TOTAL_VFS: -3}}
+        )
+        libnmstate.validator.validate(default_data)
+
+        with pytest.raises(js.ValidationError) as err:
+            libnmstate.validator.validate(default_data)
+
+        assert '-3 is lower than the minimum of 0' in err.value.args[0]
```

Comment:
wondering if you can't use the pytest.raises(... **match**) keyword here instead of this (IMHO) ugly assert.

100 % optional, I just think it has potential to look better.

Reply:
It looks much better. Thanks!